testdata/instrumenter/BinaryExpr.gobco

```diff
@@ -27,84 +27,66 @@ func binaryExpr(i int, a bool, b bool, c bool) {
 	//
 	// Also, gobco only looks at the parse tree without any type resolution.
 	// Therefore it cannot decide whether a variable is boolean or not.
-	both := GobcoCover(2, a) && GobcoCover(3, b)
-	either := GobcoCover(4, a) || GobcoCover(5, b)
+	both := GobcoCover(2, a && b)
```

Comment:
Does the condition `a && b` in this assignment strictly conform to the C1 definition? There's no explicit `then` or `else` branch here, other than the ones in the condition itself.

Reply:
in this case, it denotes Decision coverage. 😵
I confused between decision sand branch. I got clarified now. So  It should not be covered. right ?

If so, this will work
> My initial suggestion was to just remove the UnaryExpr and BinaryExpr cases. Didn't that work well enough?

Reply:
I'd say that in branch coverage mode, this code should not be instrumented as it is not part of an explicit `if` or `for` or `switch` statement.